src/storage/v2/durability/durability.cpp

```diff
@@ -31,10 +31,12 @@
 #include "storage/v2/inmemory/label_index.hpp"
 #include "storage/v2/inmemory/label_property_index.hpp"
 #include "storage/v2/inmemory/unique_constraints.hpp"
+#include "storage/v2/name_id_mapper.hpp"
 #include "utils/event_histogram.hpp"
 #include "utils/logging.hpp"
 #include "utils/memory_tracker.hpp"
 #include "utils/message.hpp"
+#include "utils/string.hpp"
```

Comment:
You probably don't need either of these includes?

Reply:
probably not, I can't recall why it was added. Maybe just automatically.